recipes/shorah/meta.yaml

```diff
@@ -0,0 +1,35 @@
+package:
+  name: shorah
+  version: "1.0.0"
+
+build:
+  number: 0
+
+source:
+  fn: shorah-1.0.0.tar.bz2
+  url: https://github.com/cbg-ethz/shorah/releases/download/v1.0.0/shorah-1.0.0.tar.bz2
+  md5: f0d057d3947b533ef4308458de9bfa8d
+
+requirements:
+  build:
+   - gcc # [linux]
+   - llvm # [osx]
+   - python >=2.7,<3
```

Comment:
@bgruening have a look at the log, without python at build time the configure script bombs out trying to byte-compile the python modules.